packages/blaze/domrange.js

```diff
@@ -1,24 +1,26 @@
+/* global Blaze */
+/* eslint-disable import/no-unresolved, no-global-assign, no-param-reassign, no-use-before-define */
 
 // A constant empty array (frozen if the JS engine supports it).
-var _emptyArray = Object.freeze ? Object.freeze([]) : [];
+const _emptyArray = Object.freeze ? Object.freeze([]) : [];
 
 // `[new] Blaze._DOMRange([nodeAndRangeArray])`
 //
 // A DOMRange consists of an array of consecutive nodes and DOMRanges,
 // which may be replaced at any time with a new array.  If the DOMRange
 // has been attached to the DOM at some location, then updating
 // the array will cause the DOM to be updated at that location.
+// eslint-disable-next-line consistent-return
 Blaze._DOMRange = function (nodeAndRangeArray) {
```

Comment:
Can you make `DOMRange` an ES6 `class`? The fmethods that are not added using `.prototype` should then be `static`